### ISSUE TITLE
Provide API to open/close a popup?

### ISSUE THREAD

**Issue by RaedsLab** [intljusticemission/react-big-calendar#803]
#### Do you want to request a _feature_ or report a _bug_?

I believe this to be a bug.  

#### What's the current behavior?

Popup closes whenever I interact with other components, this breaks my workflow.

![bug_popup](https://user-images.githubusercontent.com/1442690/38740887-e3e2b82e-3f38-11e8-95fd-3b5b6d388b4e.gif)

#### What's the expected behavior?

Being able to set if a popup is open or not. Or at least have an option to close the popup manually?

**Comment by arecvlohe** [comment]
@RaedsLab This seems like it would be helpful. Can you submit a PR?

**Comment by mikeyc7m** [comment]
+1

**Comment by drewandrew** [comment]
+1

**Comment by prescottie** [comment]
+1

**Comment by jquense** [comment]
PR's welcome!

**Comment by jquense** [comment]
@RaedsLab sorry looking at your actual example here. Can you just move the popup component you're using outside the Event component?

**Comment by nnupoor** [comment]
So I came across the same issue, where when clicking on my custom-popover, the show-more popover was closing. I think the reason being, the custom-popover element was nested inside the rbc-overlay popover. 
To counter this, I added a click handler on my custom-popover topmost  div with 
    `event.stopPropagation();
     event.nativeEvent.stopImmediatePropagation();`
 to stop the event from bubbling over to the rbc-popover. 
Hope this helps someone else facing this issue.
